lib/Elastica/Type.php

```diff
@@ -299,6 +299,22 @@ public function setMapping($mapping)
     }
 
     /**
+     * Sets value type mapping for this type with timestamp .
+     *
+     * @param \Elastica\Type\Mapping|array $mapping Elastica\Type\MappingType object or property array with all mappings
+     *
+     * @return \Elastica\Response
+     */
+    public function setMappingWithTimeStamp($mapping,$timestamp)
```

Comment:
Disagree. _timestamp it is a mapping this. There is no reason to introduce it in Type. And what about another magic mapping fields like _size, _index... etc.? Also add as param?